.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI
+
+on:
+    pull_request: {}
+    push: { branches: [ master, "1.2.x" ] }
```

Comment:
is the second one a regex pattern? don't we want to also build `1.1.x` branch, and in the future build `1.3.x` branch without having to touch this file ?

Reply:
no, it is a concrete value ("1.2.x"). GHA support patterns, I can change to to a pattern if you wish :)